bookserver/config.py

```diff
@@ -26,6 +27,13 @@
 
 # Settings
 # ========
+# Define the possible bookserver configurations. The values assigned must  be strings, since Pydantic will match these with environment variables.
+class Config(Enum):
```

Comment:
I think config was an ill conceived name on my part. I think maybe we should use FAPIConfig or something...  We use `WEB2PY_CONFIG` for the web2py server.  Using config here risks confusion with the Config for the Pydantic models and is just generally overused.

Reply:
How about `BOOK_SRV_CONFIG`? I agree that config is too generic.

Reply:
I'm good with `BOOK_SRV_CONFIG`